reconcile/templating/renderer.py

```diff
@@ -136,19 +142,17 @@ def __init__(self, local_path: str, vcs: VCS, mr_manager: MergeRequestManager):
 
     def write(self, outputs: list[TemplateOutput]) -> None:
         self.mr_manager.housekeeping()
+
+        if any([o.input.enable_auto_approval for o in outputs]):
+            auto_approved = [o for o in outputs if o.auto_approved]
+            if auto_approved:
+                self.mr_manager.create_merge_request(auto_approved)
+                return
```

Comment:
Something is fishy here. What about all not auto-approved outputs? Is the `return` correct?

Reply:
Not auto approved MRs are created after all auto approved ones are created

Reply:
(For one collection of course)

Reply:
ok